tests/integration/backup_tests/test_backups.py

```diff
@@ -103,7 +115,7 @@ async def test_build_and_deploy(ops_test: OpsTest) -> None:
 @pytest.mark.abort_on_fail
 async def test_blocked_incorrect_creds(ops_test: OpsTest) -> None:
     """Verifies that the charm goes into blocked status when s3 creds are incorrect."""
-    db_app_name = await helpers.app_name(ops_test)
+    db_app_name = await get_app_name(ops_test) or DATABASE_APP_NAME
```

Comment:
same here, and for all other `or DATABASE_APP_NAME` in this PR

Reply:
done